sdk/core/Azure.Core.Experimental/tests/LowLevelClient/PetStoreClient.cs

```diff
@@ -139,7 +162,29 @@ private HttpMessage CreateGetPetRequest(string id, RequestOptions options = null
             uri.AppendPath(id, true);
             request.Uri = uri;
             request.Headers.Add("Accept", "application/json, text/json");
+            message.ResponseClassifier = _responseClassifierCache["GetPet"];
             return message;
         }
+
+        private class GetPetResponseClassifier : ResponseClassifier
```

Comment:
Generator creates one of these per unique status code/response map, and names it by the name of the first method in the swagger that uses it.

Reply:
Update: Names it according to the status codes that are considered successful.